apps/contact/views.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib import messages
+from django.core.mail import EmailMessage
+from django.shortcuts import redirect, render_to_response
+from django.template import RequestContext
+
+from apps.contact.forms import ContactForm
+
+
+# Index page
+def index(request):
+    context = {'form': ContactForm}
+    return render_to_response('contact/index.html', context, context_instance=RequestContext(request))
+
+
+def contact_submit(request):
+    if request.method == 'POST':
+        form = ContactForm(request.POST)
+        if form.is_valid():
+            if form.cleaned_data['contact_receiver'] == '1':
+                choice = ['hovedstyret@online.ntnu.no']
+            else:
+                choice = ['dotkom@online.ntnu.no']
```

Comment:
This could be imported based on the list defined in `forms.py`, so that a change to one does not require a change to the other.

Likewise, by changing the structure of the list in `forms.py`, it's possible to do this matching automatically if more emails are to be added in the future.